packages/libs/react-ui/src/components/Typography/Text/Text.tsx

```diff
@@ -2,51 +2,76 @@ import cn from 'classnames';
 import type { FC } from 'react';
 import React from 'react';
 import {
-  boldClass,
-  colorVariant,
-  elementVariant,
-  fontVariant,
-  sizeVariant,
-  transformVariant,
-} from './Text.css';
+  bodyBaseBold,
+  bodyBaseRegular,
+  bodySmallBold,
+  bodySmallRegular,
+  bodySmallestBold,
+  bodySmallestRegular,
+  codeBaseBold,
+  codeBaseRegular,
+  codeSmallBold,
+  codeSmallRegular,
+  codeSmallestBold,
+  codeSmallestRegular,
+} from '../../../styles';
+import { colorVariants, transformVariants } from '../typography.css';
+
+// eslint-disable-next-line @kadena-dev/typedef-var
+export const TEXT_ELEMENTS = ['p', 'span', 'code'] as const;
+export type TextElementType = (typeof TEXT_ELEMENTS)[number];
+
+type TextVariant = 'small' | 'smallest' | 'base';
+function getFontClass(
+  variant: TextVariant,
```

Comment:
because the size infers it only changes the font size but it is not true since it  also changes other font properties, so it is a typography variant not only a size